infra.lvm_tools/shrink_lv/roles/shrink_lv/files/shrink-start.sh

```diff
@@ -13,8 +12,14 @@ disable_lvm_lock(){
     mv "$tmpfile" /etc/lvm/lvm.conf
 }
 
+activate_volume_groups(){
+    for vg in `/usr/sbin/lvm vgs --noheadings | /usr/bin/awk '{print $1}'`; do
```

Comment:
you could save yourself the awk command if you specify the output:
```
[root@localhost ~]# lvm vgs -o name --noheading 2>/dev/null
  rhel
```

I suggest to redirect `stderr` in case it shows warnings, but it should not affect the value coming from `stdout`.

Reply:
That's great, thanks. I've updated it

Reply:
/lgtm